### DISCUSSION TITLE
Error using WinPython 3.10

### DISCUSSION THREAD

**Discussion by alex-g-axion** [winpython/winpython#1065] [Q&A]
Hello!

I am getting an error trying to run a simple python program from Spyder IDE.  

```
import clr
print("Hello world")
```

When I try to run it I get this error:

> Traceback (most recent call last):
> 
>   File "c:\apps\wpy64-31020\python-3.10.2.amd64\lib\site-packages\clr.py", line 6, in <module>
>     load()
> 
>   File "C:\apps\WPy64-31020\python-3.10.2.amd64\lib\site-packages\pythonnet\__init__.py", line 44, in load
>     raise RuntimeError("Failed to initialize Python.Runtime.dll")
> 
> RuntimeError: Failed to initialize Python.Runtime.dll
> 
> 
> Failed to initialize pythonnet: System.InvalidOperationException: This property must be set before runtime is initialized
>    at Python.Runtime.Runtime.set_PythonDLL(String value)
>    at Python.Runtime.Loader.Initialize(IntPtr data, Int32 size)
>    at Python.Runtime.Runtime.set_PythonDLL(String value)
>    at Python.Runtime.Loader.Initialize(IntPtr data, Int32 size)

I am using winpython 3.10.2 on Windows 11.

What am I doing wrong?

**Comment by nmarc** [comment]
you have to Install a Visual Studio Redistributable package.

alex-g-axion ***@***.***> schrieb am Mo., 4. Apr. 2022, 20:15:

**Reply by alex-g-axion** [reply]
Which version?  I do have VS 2015-2022, VS 2012, VS 2010, and VS2008 installed.

**Comment by stonebig** [comment]
works for me 
![image](https://user-images.githubusercontent.com/4312421/161629838-722a2be0-06bd-45ca-9390-2d292dec2de4.png)

so
(*) For recent WinPython, Windows 8..10 users may have to install missing system DLL [Microsoft Visual C++ Redistributable for Visual Studio 2017..2019](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads) (vc_redist_x86.exe for WinPython 32bit, vc_redist_x64.exe for Winpython 64bit)
